src/types.js

```diff
@@ -243,7 +243,7 @@ export const arraySerializer = function arraySerializer(xs, serializer) {
   if (Array.isArray(first) && !first.type)
     return '{' + xs.map(x => arraySerializer(x, serializer)).join(',') + '}'
 
-  return '{' + xs.map(x =>
+  return '{' + xs.map(x => x == null ? 'null' :
```

Comment:
Doesn't this library only use strict equality comparisons?

Reply:
Yes, this isn't going in.. I've already got a fix for this in the works, so closing the PR

Reply:
You use whatever equality comparison is needed, this fix works great